test/index.js

```diff
@@ -473,6 +473,8 @@ t.test('miscellaneous headers', t => {
       t.strictSame(ua, ['agent of use'], 'UA set from options'))
     .matchHeader('npm-command', cmd =>
       t.strictSame(cmd, ['hello-world'], 'command set from options'))
+    .matchHeader('npm-use-webauthn', useWebauthn =>
+      t.strictSame(useWebauthn, ['true'], 'use-webauthn set from options'))
```

Comment:
This will be set explicitly to `false` in requests that don't have the config value, right? Let's add an assertion to one of the other tests so then we're 100% sure what the state is every time.